PovertyInfo/src/stateinfo/PrintPovertyInfo.java

```diff
@@ -3,27 +3,40 @@
 import java.io.BufferedReader;
 import java.io.FileReader;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.io.IOException;
 
 public class PrintPovertyInfo {
 	
-	public static String getMinimumChildPovertyRecord() throws IOException {
-		BufferedReader abc = new BufferedReader(new FileReader("support/povWA.txt"));
+	private static final Map<String, String> stateFiles;
+	static
+	{
+		stateFiles = new HashMap<String, String>();
+		stateFiles.put("Washington", "support/povWA.txt");
+		stateFiles.put("South Dakota", "support/povSD.txt");
+	}
+	
+	public static String getMinimumChildPovertyRecord(String state) throws IOException {
+		BufferedReader br = new BufferedReader(new FileReader(stateFiles.get(state)));
 		List<String> list = new ArrayList<String>();
 		String str;
 		
-		while((str = abc.readLine()) != null){
+		while((str = br.readLine()) != null){
 		    list.add(str);
 		}
 		
-		abc.close();
+		br.close();
 		return list.get(0);
 	}
 	
 	public static void main(String[] args) throws IOException {
```

Comment:
I usually recommend having the `main` method in a separate file from the rest of the code, so consider separating this out